app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsFragmentHelper.kt

```diff
@@ -231,47 +232,72 @@ fun FullDetailsFragment.populatePreviousButton() {
 	}
 }
 
-fun FullDetailsFragment.resumePlayback() {
+fun FullDetailsFragment.getNextUpEpisode(callback: (nextUpEpisode: BaseItemDto?) -> Unit) = lifecycleScope.launch {
```

Comment:
the callback is never called when the request fails. I think it would be better to call the callback with no next episode instead of showing the toast.